models/oakland_candidate.rb

```diff
@@ -30,6 +30,29 @@ def as_json(options = nil)
       last_name: last_name,
       ballot_item: office_election.id,
       office_election: office_election.id,
+
+      # contribution data
+      supporting_contribution_data: {
+        contributions_received: calculation(:total_contributions).try(:to_f),
+        total_contributions: calculation(:total_contributions).try(:to_f),
+        total_expenditures: calculation(:total_expenditures).try(:to_f),
+        total_loans_received: calculation(:total_loans_received).try(:to_f),
+        contributions_by_type: calculation(:contributions_by_type) || {},
+        expenditures_by_type: calculation(:expenditures_by_type) || {},
+      },
+      opposing_contribution_data: {
```

Comment:
Ah, I got an answer, it does get reported. A committee can exist for the purpose of defeating a candidate and will report expenditures as opposing the candidate. Carry on.